documentation/modules/overview/con-key-features-kafka-connect.adoc

```diff
@@ -0,0 +1,141 @@
+A Kafka Connect cluster might operate source and sink connectors at the same time.
+Workers are running in distributed mode in the cluster.
+Workers can run one or more tasks for more than one connector instance.
+
+.Source connector streaming data to Kafka
+image:overview/kafka-concepts-source-connector.png[Kafka Connect source connector worker interaction in distributed mode]
+
+. A plugin provides the implementation artifacts for the source connector
+. A single worker initiates the source connector instance
+. The source connector creates the tasks to stream data
+. Tasks run in parallel to poll the external data system and return records
+. Transforms adjust the records, such as filtering or relabelling them
+. Converters put the records into a format suitable for Kafka
+. The source connector is managed using KafkaConnectors or the Kafka Connect API
+
+The following sink connector diagram shows the process flow when streaming data from Kafka to an external data system.
+
+.Sink connector streaming data from Kafka
+image:overview/kafka-concepts-sink-connector.png[Kafka Connect sink connector worker interaction in distributed mode]
+
+. A plugin provides the implementation artifacts for the sink connector
+. A single worker initiates the sink connector instance
+. The sink connector creates the tasks to stream data
+. Tasks run in parallel to poll Kafka and return records
+. Converters put the records into a format suitable for the external data system
+. Transforms adjust the records, such as filtering or relabelling them
+. The sink connector is managed using KafkaConnectors or the Kafka Connect API
+
+== Tasks
+
+Data transfer orchestrated by the Kafka Connect runtime is split into tasks that run in parallel.
+A task is started using the configuration supplied by a connector instance.
+Kafka Connect distributes tasks to workers.
+
+* A source connector task polls the external data system and returns a list of records that a worker sends to the Kafka brokers.
+* A sink connector task receives Kafka records from a worker for writing to the external data system.
+
+If a task fails, it can be restarted through the Kafka Connect API.
+
+For sink connectors, the number of tasks created relates to the number of partitions being consumed.
+For source connectors, how the source data is partitioned is defined by the connector.
+You can control the maximum number of tasks that can run in parallel by setting `tasksMax` in the connector configuration.
+
+== Workers
+
+Workers store the initial connector configuration deployed to the Kafka Connect cluster.
+They also start connectors and their tasks.
+
+* Source connector workers read events from an external data system and use converters to construct key and value pairs to store in Kafka.
+* Sink connector workers read records from Kafka and use converters to construct records to pass into an external data system.
```

Comment:
I don't think we need to explain this again. This section is about workers. It's enough to say they run connectors and tasks. What that means is explained in the sections on connectors and tasks.

Reply:
Removed